EventFilter/Utilities/plugins/RecoEventOutputModuleForFU.h

```diff
@@ -72,6 +72,7 @@ namespace evf {
 
   template<typename Consumer>
   RecoEventOutputModuleForFU<Consumer>::RecoEventOutputModuleForFU(edm::ParameterSet const& ps) :
+    edm::one::OutputModuleBase::OutputModuleBase(ps),
```

Comment:
Looks like it is an unfortunate side-effect of the design because of the use of multiple inheritance and the need to pass the ParameterSet down the inheritance chain.